contracts/core/IBCKeeper.sol

```diff
@@ -1,25 +1,18 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.6.8;
+pragma solidity ^0.8.9;
 pragma experimental ABIEncoderV2;
```

Comment:
This pragma is no longer needed because ABIEncoderV2 becomes the default encoder for solidity v0.8.
The same applies to other files as well.